homeassistant/loader.py

```diff
@@ -250,6 +250,26 @@ async def async_get_config_flows(
     return flows
 
 
+async def async_get_config_flows_v2(
+    hass: HomeAssistant,
+) -> dict[str, Any]:
+    """Return cached list of config flows."""
+    # pylint: disable=import-outside-toplevel
+    from .generated.config_flows_v2 import FLOWS
```

Comment:
Since we only need this file when adding an integration, let's load it from a JSON file so Python doesn't keep it in memory?

Reply:
Sure, fixed in 440f13d31183d97a1e6d0534f9e3e96d5d5f60d7